t/t1000-kvs.t

```diff
@@ -438,6 +438,37 @@ test_expect_success 'kvs: zero-length value does not cause ls -FR to fail' '
 '
 
 #
+# treeobj tests
+#
+test_expect_success 'kvs: treeobj of all types handled by get --treeobj' '
+	flux kvs unlink -Rf $DIR &&
+	flux kvs put $DIR.val=hello &&
+	flux kvs put $DIR.valref=$(seq -s 1 100) &&
+	flux kvs mkdir $DIR.dirref &&
+	flux kvs link foo $DIR.symlink &&
+	flux kvs get --treeobj $DIR.val &&
```

Comment:
perhaps ```| grep "dir"``` and equivalents should be added to make sure the right treeobj is being returned

Reply:
Sure, that's a good improvement.